src/lib/validationSchemas/createEventSchema.js

```diff
@@ -0,0 +1,75 @@
+import { z } from 'zod';
+import { minMaxString } from './utils';
+
+export const CreateEventSchema = z
```

Comment:
🟠 how do you think, is it makes sense to move all admin-related validation schemas to `admin` folder?

Reply:
Makes sense, I'll move it